code/modules/surgery/organs/tongue.dm

```diff
@@ -28,7 +28,8 @@
 		/datum/language/slime,
 		/datum/language/sylvan,
 		/datum/language/terrum,
-		/datum/language/uncommon))
+		/datum/language/uncommon,
+		/datum/language/vampiric)) // austation -- Adding Bloodsuckers by Coolsurf6: Adding the bloodsucker tongues so they can speak their language
```

Comment:
you can modularize out the languages. see yaki's old language PR for reference (has been reverted because bee ported them) #1712